package.json

```diff
@@ -218,10 +218,10 @@
     "cross-env": "^7.0.0",
     "css-loader": "^3.6.0",
     "detect-port": "^1.3.0",
-    "electron": "9.1.0",
-    "electron-builder": "^22.7.0",
-    "electron-devtools-installer": "^3.1.1",
-    "electron-rebuild": "^1.11.0",
+    "electron": "7.1.13",
```

Comment:
Let's try to see if we can get this working with `electron@^8`. 7 has a lot of bugs